src/models/hooks/objectiveResource.js

```diff
@@ -49,24 +49,26 @@ const propagateCreateToTemplate = async (sequelize, instance, options) => {
   if (objective
     && objective.objectiveTemplateId !== null
     && objective.objectiveTemplate.creationMethod === AUTOMATIC_CREATION) {
-    const [otr] = await sequelize.models.ObjectiveTemplateResource.findOrCreate({
+    const [otr, wasCreated] = await sequelize.models.ObjectiveTemplateResource.findOrCreate({
```

Comment:
is there an opportunity for some performance benefit by moving away from the ```findOrCreate```?

Reply:
I will look into it